tests/test_ercot_allbusrt.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
You should be able to add a new function to tests/test_ercot.py modeled on the ```test_request_report_rt5m_lmp``` function already there.  You can also see the structure of our test suite in that file